bin/crew

```diff
@@ -1243,29 +1243,53 @@ def install_package(pkgdir)
 
     rsync_available = true if File.exist?("#{CREW_PREFIX}/bin/rsync") and `#{CREW_PREFIX}/bin/rsync --version`.include? 'rsync  version'
     puts 'rsync is not working. Please (re)install it with \'crew remove musl_zstd zstd ; crew install musl_zstd ; crew install rsync\''.lightred unless rsync_available or @pkg.name == 'rsync'
+    
+    # Create filelists for rsync and tar to use. Use standard paths since 
+    # builds always use these paths.
+    FileUtils.touch %w[
+      dflist.rsync.home dflist.rsync.usrlocal dflist.tar.home 
+      dflist.tar.usrlocal
+    ]
+    # Create filelists for rsync to use.
+    system "grep -s \"^/home/chronos/user\" #{CREW_META_PATH + @pkg.name + '.directorylist'} | cut -c 19- > dflist.rsync.home"
+    system "grep -s \"^/home/chronos/user\" #{CREW_META_PATH + @pkg.name + '.filelist'} | cut -c 19- >> dflist.rsync.home"
+    
+    system "grep -s \"^/usr/local\" #{CREW_META_PATH + @pkg.name + '.directorylist'} | cut -c 11- > dflist.rsync.usrlocal"
+    system "grep -s \"^/usr/local\" #{CREW_META_PATH + @pkg.name + '.filelist'} | cut -c 11- >> dflist.rsync.usrlocal"
```

Comment:
Abstract with `#{CREW_PREFIX}`?

Reply:
Same why as before. The crew prefix in built packages should always be `/usr/local`, otherwise installing binary packages will break, right?

We could specify the home and crew prefix used in a binary package in a file in the package though, and use that if it is there...